team-bimmer/ribbit
# Decouple linking/unlinking user depenencies from server

### **What?**
Linking user dependencies has now been decoupled from the server.

### **Why?**
Separation of concerns. Also our dependencies should be installed before the server is initiated, and this is a necessary change before we make the shift to a plugin architecture.

### **How to test?**
Since this branch is based on pull request #40 , please ensure that you are:
1.) Using the **'redux-demo' app** and that
2.) You have **linked the CSS files in the App component**.

Running ribbit build should generate the static files without issue even though we don't have babel-loader installed.

### React, redux, react-redux, webpack, and webpack-cli can now be npm-uninstalled from Ribbit!
Simply add them to the dependencies array in the demo app's ribbit config and ribbit build should still work without issues! =D
